client/state/test/initial-state.js

```diff
@@ -0,0 +1,148 @@
+require( 'lib/react-test-env-setup' )();
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import sinon from 'sinon';
+import localforage from 'localforage';
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+import createReduxStoreFromPersistedInitialState, { MAX_AGE } from 'state/initial-state';
+
+describe( 'initial-state', () => {
+	describe( 'createReduxStoreFromPersistedInitialState', () => {
+		describe( 'persist-redux disabled', () => {
+			describe( 'with recently persisted data and initial server data', () => {
+				var configStub,
+					consoleSpy,
+					state,
+					serverState = { ui: { section: 'hello' } };
+				before( ( done ) => {
+					window.initialReduxState = serverState;
+					configStub = sinon.stub( config, 'isEnabled' );
+					configStub.withArgs( 'persist-redux' ).returns( false );
+					consoleSpy = sinon.spy( console, 'error' );
```

Comment:
Definitely not necessary to change, though as an FYI, you don't need to assign the spy to a variable. Instead:

```js
sinon.spy( console, 'error' );
expect( console.error.called ).to.equal( false );
console.error.restore();
```